File: PaperBook.java
package ua.lviv.iot.algo.part1.Book;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString(callSuper = true)
public class PaperBook extends Book {
    private int pageCount;
    private int widthInMm;
    private int heightInMm;

    public PaperBook(String title, String author, String publisher, int year, String genre, int pageCount, int widthInMm, int heightInMm) {
        setTitle(title);
        setAuthor(author);
        setPublisher(publisher);
        setYear(year);
        setGenre(genre);
        setPageCount(pageCount);
        setWidthInMm(widthInMm);
        setHeightInMm(heightInMm);
    }
    @Override
    public int getPagesCount() {
        return pageCount;
    }
}

File: PaperBook.py
from models.Book import Book

class PaperBook(Book):
    def __init__(self, title, author, publisher, year, genre, page_count, width_in_mm, height_in_mm):
        super().__init__(title, author, publisher, year, genre)
        self.page_count = page_count
        self.width_in_mm = width_in_mm
        self.height_in_mm = height_in_mm

    def get_pages_count(self):
        return self.page_count